waggle-dance-rest/pom.xml

```diff
@@ -80,6 +80,12 @@
       <groupId>org.skyscreamer</groupId>
       <artifactId>jsonassert</artifactId>
       <scope>test</scope>
+      <exclusions>
```

Comment:
Do you need to do this in order to prevent a conflicting version of Jackson being pulled in?

Reply:
The rest controller test prints a warning about two similar class in two different jars on the classpath. It Adviced to remove one for deterministic test sake :). I don't need to do this for the fix.